h2/src/docsrc/help/help.csv

```diff
@@ -840,7 +840,7 @@ CREATE INDEX IDXNAME ON TEST(NAME)
 @h2@ CREATE [ FORCE ] [ [ GLOBAL | LOCAL ] TEMPORARY ]
 @h2@ LINKED TABLE [ IF NOT EXISTS ]
 @h2@ [schemaName.]tableName ( driverString, urlString, userString, passwordString,
-@h2@ [ originalSchemaString, ] @h2@ originalTableString ) [ EMIT UPDATES | READONLY ]
+@h2@ [ originalSchemaString, ] @h2@ originalTableString ) [ EMIT UPDATES | READONLY ] [FETCH_SIZE numberOfRows]
```

Comment:
This line seems to be very long for PDF, I think it would be better to move last clauses to the new line
```
@h2@ [ EMIT UPDATES | READONLY ] [ FETCH_SIZE sizeInt ]
```
Name of parameter should have its value type (`Int`, `Long`, `String`, etc.) at the end; these names are parsed to add a reference to the grammar of literal.

Reply:
done